package.json

```
@@ -1,10 +1,10 @@
 {
   "name": "lyra",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "description": "Root package of Lyra",
   "private": true,
   "engines": {
-    "node": ">=18.17"
+    "node": ">=23.8"
```

Amer updated main to Node 22 in 9ce82da4aa8c6f823cb24ceba68b89bfc751fc08

I suggest resolving that conflict by accepting Amer's version and then updating package-lock with `npm install`

```suggestion
    "node": ">=22"
```